libs/entitlement/tools/common/src/main/java/module-info.java

```diff
@@ -0,0 +1,3 @@
+module org.elasticsearch.libs.entitlement.tools.common {
```

Comment:
Question for reviewers: is it worth having these things as modules? I have a separate commit "adding modularity", I can revert it if we think is not worth the hassle.

Reply:
Since this is just a build tool, it's probably not necessary.

Reply:
I am with you. 
Better revert the change then? Or since I have modularized it already should I just leave it this way? Wdyt?

Reply:
If you leave it, I suggest at least matching the pattern of the module name with other modules in ES (eg `elasticsearch.something`